govcd/lbappprofile_test.go

```diff
@@ -71,6 +71,17 @@ func (vcd *TestVCD) Test_LBAppProfile(check *C) {
 
 	check.Assert(createdLbAppProfile.Template, Equals, lbAppProfileConfig.Template)
 
+	// Test that we can extract a list of LB app profiles, and that one of them is the profile we have got when searching by name
+	lbAppProfiles, err := edge.GetLbAppProfiles()
+	check.Assert(err, IsNil)
```

Comment:
check that `lbAppProfiles` is not nill is missing

Reply:
done